pkg/products/bamboo/secrets.tf

```diff
@@ -0,0 +1,43 @@
+################################################################################
+# Kubernetes secret to store db credential
+################################################################################
+resource "kubernetes_secret" "rds_secret" {
+  metadata {
+    name      = "${local.product_name}-db-cred"
+    namespace = kubernetes_namespace.bamboo.metadata[0].name
+  }
+
+  data = {
+    username = module.database.rds_master_username
+    password = module.database.rds_master_password
+  }
+}
+
+################################################################################
+# Kubernetes secret to store license
+################################################################################
+resource "kubernetes_secret" "license_secret" {
+  metadata {
+    name = "${local.product_name}-license"
+  }
+
+  data = {
+    license = var.license
+  }
+}
+
+################################################################################
+# Kubernetes secret to store system admin credentials
+################################################################################
+resource "kubernetes_secret" "admin_secret" {
```

Comment:
Also this one should go to they bamboo namespace. Each product may have different admin info.